updater/release_checker.go

```diff
@@ -90,7 +93,28 @@ func (c *ReleaseChecker) update(ctx context.Context) error {
 		return err
 	}
 
-	if latest == c.current {
+	if c.current == latest {
```

Comment:
I think a local value should come first for binary operators:
```suggestion
	if latest == c.current {
```